tools/validators/instance_validator/tests/fake_instances/GOOD/translation_units_and_states.yaml

```diff
@@ -12,26 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-CHWS_WDT-17-GUID:
-  type: HVAC/CHWS_WDT
-  code: CHWS_WDT-17
+# Building
+US-SEA-BLDG1-GUID:
+  type: FACILITIES/BUILDING
+  code: US-SEA-BLDG1
+
+FAN-17-GUID:
+  type: HVAC/FAN_SS_VSC
+  code: FAN-17
   cloud_device_id: "foobar"
   translation:
-    return_water_temperature_sensor:
-      present_value: "points.return_water_temperature_sensor.present_value"
-      units:
-        key: "points.return_water_temperature_sensor.units"
-        values:
-          degrees_celsius: "degC"
-          degrees_fahrenheit: "degF"
-    supply_water_temperature_sensor:
-      present_value: "points.supply_water_temperature_sensor.present_value"
+    run_command:
+      present_value: points.run_command.present_value
+      states:
+        OFF: false
+        ON: true
+    run_status:
+      present_value: points.run_status.present_value
+      states:
+        OFF: 0
+        ON: 1
+    speed_percentage_command:
+      present_value: points.supply_fan_speed_percentage_command.present_value
       units:
-        key: "points.supply_water_temperature_sensor.units"
```

Comment:
I believe this is the correct form?

Reply:
Done.

Reply:
remove "pointset" and add quotes around all the strings that start with "points"